collector-state/README.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: pattern
+title: collector-state
+folder: collector-state
+permalink: /patterns/collector-state/
+categories: Behavioral
+---

```

Comment:
Add `language: en`

Reply:
I can't find a good match from the existing tags, but it would be nice to have one..